src/page-response-impact/responseChartData.tsx

```diff
@@ -51,11 +56,16 @@ export function getCurveInputs(modelInputs: EpidemicModelState[]) {
 }
 
 function originalEpidemicModelInputs(systemWideData: SystemWideData) {
-  const { staffPopulation, incarceratedPopulation } = systemWideData;
+  const {
+    staffPopulation,
+    staffCases,
+    incarceratedPopulation,
+    incarceratedCases,
+  } = systemWideData;
   return {
-    staffCases: 1,
+    staffCases: staffCases,
     staffPopulation: staffPopulation,
-    ageUnknownCases: 1,
+    ageUnknownCases: incarceratedCases,
```

Comment:
@macfarlandian Question for you. Do the curve charts treat the different cases by age group inputs differently? So, would setting input `age20Cases=15, age85Cases=3` as inputs give a different result than `ageUnknownCases=18`? For our original projection inputs I'm just using the sum of incarcerated cases and assigning it to `ageUnknownCases` here but I'm starting to think that this approach is not going to produce a fair comparison to the actual scenario model inputs.

Reply:
yes they do .... they have different fatality rates and soon they will have different hospitalization rates as well.

Short of actually getting the age breakdowns for the "original" projection though, putting them all as age unknown is the best bet, since that uses rates that are averaged across all brackets. it's definitely going to be less accurate but it may be the best we can do with the data we have?